backend/main.go

```diff
@@ -100,7 +110,12 @@ func main() {
 		log.Fatal().Err(err).Msg("failed to migrate database")
 	}
 
-	processor := &Processor{}
+	processor := &Processor{
+		telegramAlertProvider: NewTelegramAlertProvider(TelegramProviderConfig{
+			Url:    telegramUrl,
+			ChatID: telegramChatID,
+		}),
+	}
```

Comment:
Instead, this should be array of available alerter. But we'll work on refactoring that later.